lib/rules/declaration-property-value-no-unknown/__tests__/index.js

```diff
@@ -0,0 +1,246 @@
+'use strict';
+
+const { messages, ruleName } = require('..');
+
+testRule({
+	ruleName,
+	config: true,
+
+	accept: [
+		{
+			code: 'a { top: 0; }',
+		},
+		{
+			code: 'a { margin: auto 10em; }',
+		},
+		{
+			code: 'a { margin-inline: calc(100% - 10px); }',
+		},
+		{
+			code: 'a { top: var(--foo); }',
+		},
+		{
+			code: 'a { foo: 1px; }',
+		},
+		{
+			code: 'a { --foo: 1px; }',
+		},
+		{
+			code: 'a { color: lightgrey; }',
+		},
+		{
+			code: 'a { color: lightgray; }',
+		},
+		{
+			code: 'a { display: -moz-inline-stack; }',
+		},
+		{
+			code: 'a { color: -moz-initial; }',
+		},
+		{
+			// TODO: CSSTree doesn't support math functions like `clamp()` yet, but the future update will support them.
+			// See https://github.com/stylelint/stylelint/pull/6511#issuecomment-1412921062
+			skip: true,
+			code: 'a { font-size: clamp(1rem, 1vw + 1rem, 3rem); }',
+		},
```

Comment:
Let's favour false negatives over positives.

If we can't use `fork()` in this instance, let's return early if [we find](https://github.com/csstree/csstree/blob/master/docs/traversal.md#findast-fn) a `clamp`, `min` or `max` function in the value.

Reply:
Thanks for sharing the nice way to find functions. Fixed via 93171400.